tests/ut/ir/statements/test_scope_stmt_hierarchy.py

```
@@ -117,91 +92,41 @@ def test_scope_stmt_hierarchy_global():
-    s_none = ir.ScopeStmt(ir.ScopeKind.InCore, _empty_body(), _span())
-    s_set = ir.ScopeStmt(
-        ir.ScopeKind.Hierarchy,
-        _empty_body(),
-        _span(),
-        level=ir.Level.HOST,
-    )
+def test_structural_equal_different_kinds():
+    """Different scope kinds (InCore vs Hierarchy) compare as unequal."""
+    s_in = ir.InCoreScopeStmt(body=_empty_body(), span=_span())
+    s_hier = ir.HierarchyScopeStmt(level=ir.Level.HOST, body=_empty_body(), span=_span())
     with pytest.raises(Exception):
```

Ruff (0.15.10)

[warning] Do not assert blind exception: `Exception`

(B017)